python/agents/utils/objects.py

```
@@ -0,0 +1,228 @@
+    active: bool
+    closed: bool
+    seconds_delay: int
+    icon: str
+    fpmm: str
+    name: str
+    description: Union[str, None] = None
+    price: float
+    tax: Union[float, None] = None
+
+
+class SimpleEvent(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)